content/en/developers/smart-contracts/how-tos/deploy-a-contract/index.md

```diff
@@ -63,7 +63,7 @@ You must have the following installed:
 - [Git](https://git-scm.com/)
 - [Yarn](https://yarnpkg.com/)
 
-You should also have an address on the Filecoin Hyperspace testnet. See the [Add to MetaMask page]({{< relref "add-to-metamask" >}}) for information on how to get an address. You also need test-FIL `tFIL` in your wallet. See the [Use a Faucet]({{< relref "use-a-faucet" >}}) page for information on how to get test funds.
+You should also have an address on the Filecoin Hyperspace testnet. See the [Add to MetaMask page]({{< relref "add-to-metamask" >}}) for information on how to get an address. You also need test-FIL `tFIL` in your wallet. See the [Use a Faucet]({{< relref "get-test-funds" >}}) page for information on how to get test funds.
```

Comment:
Can we use `get-test-tokens`? 
Tokens make more sense than funds for blockchain projects.

Reply:
Good point. I'll change that.

Reply:
Resolved in  38191d95